src/tribler/gui/tests/conftest.py

```diff
@@ -61,3 +62,25 @@ def pytest_runtest_protocol(item, log=True, nextitem=None):
     total = time.time() - pytest_start_time
     if enable_extended_logging:
         print(f' in {duration:.3f}s ({total:.1f}s in total)', end='')
+
+
+@pytest.fixture(autouse=True)
+def ensure_gc():
```

Comment:
Do we need this on the GUI test?

Reply:
Probably not, but it can help isolate GUI tests a tiny bit better